tests/bls12381-add/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
 [package]
 name = "bls12381-add-test"
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
we should not need to version tests/examples. they could just always be 0.1.0? since they aren't published or used externally

Reply:
yeah your right, Ill reset those next release